projects/js-packages/api/index.jsx

```diff
@@ -519,6 +519,16 @@ function JetpackRestApiClient( root, nonce ) {
 			getRequest( `${ apiRoot }jetpack/v4/search/pricing`, getParams )
 				.then( checkStatus )
 				.then( parseJsonResponse ),
+		fetchSocialSettings: () =>
```

Comment:
This seems like the standard pattern, but it feels a little odd to me. It's only the Social the plugin that will call this (unless it becomes needed by My Jetpack) and this change is triggering a changelog requirement to all the packages and plugins that use the API package. Do you think we should move this to the Social plugin?